build.gradle.kts

```diff
@@ -68,4 +69,8 @@ tasks {
       jvmTarget = "15"
     }
   }
+
+  test {
+    jvmArgs("--add-opens", "java.base/java.lang=ALL-UNNAMED")
```

Comment:
this is for running the tests, had to put the same change in the Dockerfile for running the app